### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by bocheraATC10** [community/community#110054] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

Hey guys. I am a beginner and I am struggling a little bit. How do I know the exact image path. Currently i uploaded an image file which contains the image that i want. So how do i insert the image into my website. Please tell me if you can.

**Comment by ItsJoshBrown** [comment, marked as answer]
If you have what you are referring to in your repo, you can do something like this. 

```HTML
<img src="/image/Meowmad-ali.png" alt="Meowmad Ali Cat">
```

Here would be the tree structure of the folders. 

This would be using relative paths, with the file path in the image folder and selecting the file name.
```
.
├── image/
│   └── Meowmad-ali.png
├── README.md
└── index.html
```

Depending on what code editor you are using, you can probably just click on the file and copy the relative file path.

**Reply by bocheraATC10** [reply]
Thank you very very much!!That really helped me a lot. Thank you. it means a lot!!!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bocheraATC10 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
